src/control/controlindicator.h

```diff
@@ -17,24 +22,24 @@ class ControlIndicator : public ControlObject {
     ControlIndicator(const ConfigKey& key);
     virtual ~ControlIndicator();
 
-    void setBlinkValue(enum BlinkValue bv);
+    void setBlinkValue(enum BlinkValue newBlinkValue);
 
   signals:
     void blinkValueChanged();
 
   private slots:
-    void slotGuiTick50ms(double cpuTime);
     void slotBlinkValueChanged();
+    void set(double value) {
+        ControlObject::set(value);
+    };
 
   private:
     void toggle(double duration);
     // set() is private, use setBlinkValue instead
```

Comment:
This comment is now misplaced. And I also don't understand it. setBlinkValue() serves a different purpose than set().

Reply:
Just delete the comment.